### DISCUSSION TITLE
Context menu question

### DISCUSSION THREAD

**Discussion by silverwings15** [tomasklaen/uosc#558] [Q&A] [answered]
I have the context menu set to open on right click/mouse2. Is it possible to make the context menu to appear at the location of the cursor when pressed, rather than always at the center of the screen?

Also, bonus question if you'll humor me since I don't know where else to ask: When using the seekbar, the video stops playing until you release the left click/mouse1. Is it possible to set it to continue to play even if mouse4 is still pressed down?

Would appreciate any help!!

**Comment by tomasklaen** [comment, marked as answer]
We can't render it in other place because we can't render outside of the window bounding box.

Pause while holding was a requested feature and it makes sense. It allows people to seek precisely the frame they want without having to click every pixel. Ignoring hold and mouse up just feels wrong and a missing feature. If you grab a handle (in this scenario time), everyone would expect the handle to be grabbed and stay under cursor, not continue moving forward.

**Reply by silverwings15** [reply]
ah well, as a recent migrate from PotPlayer i guess i just need more time to get used to how things are on MPV. really appreciate your answers!
